server/src/completions/spdx-licenses.ts

```diff
@@ -37,7 +60,27 @@ export const licenseCompletionItems: CompletionItem[] = spdxLicenses.map((licens
     kind: CompletionItemKind.Value,
     deprecated: license.isDeprecatedLicenseId === true,
     labelDetails: {
-      description: spdxLicenseSource
-    }
+      description: spdxLicenseDescription
+    },
+    documentation: 'Loading...',
+    data: {
+      source: spdxLicenseDescription,
+      payload: license
+    } satisfies CompletionItemData
   }
 ))
+
+export const getSpdxLicenseCompletionResolve = async (item: CompletionItem): Promise<CompletionItem> => {
+  try {
+    const license = item.data.payload as SpdxLicense
+    const detailsResponse = await fetch(license.detailsUrl)
```

Comment:
Is the database loaded every time completion is requested? We should load it at most once, lazy if possible (only when needed the first time)

Reply:
done

Reply:
Ok but it's not lazy.
```typescript
const spdxLicensesPath = path.join(__dirname, '../../resources/spdx-licenses.json')
const spdxLicensesFileContent = fs.readFileSync(spdxLicensesPath, 'utf8')
export const spdxLicensesCollection = JSON.parse(spdxLicensesFileContent) as SpdxLicenseCollection
```
We'd have faster startup if this was delayed and cached to the first call of `getLicenseCompletionItems()`